### ISSUE TITLE
A11y_OSvisualupdate_MiniControGallery_GenericControls_Screenreader : Using NVDA the screen reader users are unable activate the control while navigation using "Insert + arrow keys".

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/microsoft-ui-xaml#1589]
GitHubTags: #A11y_OSvisualupdate, #A11yMAS, #A11yTCS, #Win10, #DesktopApp, #WindowsUILibrary, #MAS1.3.1, #MAS2.1.1, #NVDA

**Environment Details:**
Application Name: OS visual update
Windows Version: Windows10
Screen Reader: Narrator, NVDA 2019.2

**Steps to Reproduce:**
1. Launch Mini Control Gallery App from the installer location.
2. Screen with all generic controls would appear on left side of the screen.
3. Verify that all the elements of "MiniControlGallery-Generic Controls" are accessible and MAS Compliant.

**Actual:**
When screen reader users navigate through the controls on the screen using NVDA with the “Insert + arrow keys”, then they are unable to activate controls but when they navigate using the “Tab” key they are able to activate it.

**Similar issue is observed in the below scenario:**
1. MiniControlGallery - Calender and Color Picker Controls -> Launch Mini Control Gallery App from the installer location. -> Screen with Calender and Color Picker controls would appear on right side of the screen.

**Expected:**
The screen reader users should be able to activate the control even when they are navigating using the “Insert + arrow keys”.

**User Impact:**
The screen reader users are not able to interact with the controls using NVDA then they will miss out the information that is able for the keyboard users.

**Attachment:**
[A11y_OSvisualupdate_MiniControGallery_GenericControls_Screenreader  Using NVDA the screen reader users are unable activate the control while navigation using Insert + arrow ke.pptx](https://github.com/microsoft/microsoft-ui-xaml/files/3830693/A11y_ControlUIupdatebasedonSantoriniOSvisualupdate_MiniControGallery_GenericControls_Screenreader.Using.NVDA.the.screen.reader.users.are.unable.activate.the.control.while.navigation.using.Insert.%2B.arrow.ke.pptx)

**Comment by jevansaks** [comment]
I don't know enough about NVDA, what patterns or properties are different such that Narrator works but NVDA doesn't?
